### ISSUE TITLE
List all the things that the app needs to import from Bootstrap

### ISSUE THREAD

**Issue by balinterdi** [balinterdi/rarwe-issues#15]
Page 34 (pdf version) only lists `bootstrap.css` as being imported. A whole lot more is needed for later chapters (glyphicons & the source map).

**Comment by balinterdi** [comment]
This is now  fixed and goes out in the next release. Here is what is needed:

```js
// Brocfile.js
(...)
app.import('bower_components/bootstrap/dist/css/bootstrap.css');
app.import('bower_components/bootstrap/dist/css/bootstrap.css.map', { destDir: 'assets' });
app.import('bower_components/bootstrap/dist/fonts/glyphicons-halflings-regular.eot', { destDir: 'fonts' });
app.import('bower_components/bootstrap/dist/fonts/glyphicons-halflings-regular.svg', { destDir: 'fonts' });
app.import('bower_components/bootstrap/dist/fonts/glyphicons-halflings-regular.ttf', { destDir: 'fonts' });
app.import('bower_components/bootstrap/dist/fonts/glyphicons-halflings-regular.woff', { destDir: 'fonts' });
app.import('bower_components/bootstrap/dist/fonts/glyphicons-halflings-regular.woff2', { destDir: 'fonts' });

module.exports = app.toTree();
```

**Comment by balinterdi** [comment]
Fixed for 1.11.1
